### ISSUE TITLE
Error: Can't resolve './v4/soccor.js'

### ISSUE THREAD

**Issue by codefactor** [SAP/ui5-webcomponents#4505]
### **Bug Description**
The following error occurs on story book when trying to import arbitrary icons:
```
Error: Can't resolve './v4/soccor.js' in '/SAPDevelop/xweb/mini-app-lib/node_modules/@ui5/webcomponents-icons/dist'
```
It appears there is a duplicate icon called "soccor" - this is probably supposed to be "soccer"
https://github.com/SAP/ui5-webcomponents/blob/7909b5c0de3b129f741bedb7447b707a2b7744f7/packages/icons/src/v4/SAP-icons.json#L1862

For some reason the duplicate soccor icon is not generated into v4/v5 directories.

### **Expected Behavior**
The error should not occur.

### **Steps to Reproduce**
Using webpack try to use
```
await import(`@ui5/webcomponents-icons/dist/${icon}`);
```

### **Isolated Example**
N/A

### **Context**
- UI5 Web Components version: {...}
- OS/Platform: {...}
- Browser: {...}
- Affected component: {...}

### **Log Output / Stack Trace / Screenshots**
```
{...}
```
### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: {...}
- Bussiness impact: {...}

**Comment by codefactor** [comment]
This appears related to https://github.com/SAP/ui5-webcomponents/pull/4318

**Comment by codefactor** [comment]
@vladitasev ,
Is it possible to add a unit test case that will scan all of the icons and validate that they exist in both v4 and v5, so that way you don't generate javascript files that have broken imports?

Even better, would it be possible to scan all imports in your generated package and ensure that they exist before publishing it?

**Comment by codefactor** [comment]
Here is a build script that added as a `postinstall` to my project to patch the issue while we wait for a new release:
```javascript
const path = require("path");
const fs = require("fs");
let finished = false;
let pdir = __dirname;
while (!finished) {
  const next = path.join(pdir, "..");
  finished = next === pdir;
  if (!finished) {
    pdir = next;
    const distDir = path.join(pdir, "node_modules", "@ui5", "webcomponents-icons", "dist");
    if (fs.existsSync(distDir)) {
      [".js", ".svg"].forEach((e) => {
        // There was a typo in the icons.json copied from OpenUI5, in 1.0.2 version of ui5-webcomponents
        // there is a broken link which would cause webpack to fail.
        const soccor = path.join(distDir, "v4", `soccor${e}`);
        const soccer = path.join(distDir, "v4", `soccer${e}`);
        const soccorv5 = path.join(distDir, "v5", `soccer${e}`);
        if (!fs.existsSync(soccor) && fs.existsSync(soccer) && fs.existsSync(soccorv5)) {
          fs.copyFileSync(soccer, soccor);
          console.log(`Copy from: ${soccer}\n Copy to:   ${soccor}`);
        }
      });
    }
  }
}
```

**Comment by vladitasev** [comment]
Hi @codefactor 

Sorry for that. We discovered the same issue a while ago (build script breaking).

The `soccor` icon was a typo in the Fiori icons collection which was fixed after the icons had been released. It seems the icons authors realized there was a typo and added `soccer` (correct spelling) instead of fixing the wrong one, so now 2 icons with the same content and slightly different names exist.

Initially, we deleted the wrong one and decided not to ship it at all, but then came the `v4` vs `v5` icons story, and then `soccor` reemerged as part of `v5`. This caused the error you're reporting.

Finally, we re-added `soccor` and decided to go for full parity with the official library.

Regards,
Vladi
